llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -436,6 +436,19 @@ static cl::opt<AsanDtorKind> ClOverrideDestructorKind(
                           "Use global destructors")),
     cl::init(AsanDtorKind::Invalid), cl::Hidden);
 
+// SYCL flags
+static cl::opt<bool> ClOffloadGlobals("asan-offload-globals",
```

Comment:
To differentiate from e.g. AMDGPU offload
```suggestion
static cl::opt<bool> ClSpirOffloadGlobals("asan-spir-offload-globals",
```

Reply:
I changed it to more simple string "asan-spir-globals".